databox/api/tests/MetadataManipulator/WriterTest.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Tests\MetadataManipulator;
+
+use Alchemy\ApiTest\ApiTestCase;
+use Alchemy\MetadataManipulatorBundle\MetadataManipulator;
+use PHPExiftool\Driver\Metadata\Metadata;
+use PHPExiftool\Driver\Metadata\MetadataBag;
+use PHPExiftool\Driver\TagGroup\IFD0\Artist;
+use PHPExiftool\Driver\TagGroupInterface;
+use PHPExiftool\Reader;
+use PHPExiftool\Driver\Value\Mono;
+
+
+class WriterTest extends ApiTestCase
+{
+    private ?MetadataManipulator $service = null;
+    private ?Reader $reader = null;
+
+    public static function setUpBeforeClass(): void
+    {
+        // _w will be used for write tests
+        copy(__DIR__.'/../fixtures/files/Metadata_test_file.jpg', __DIR__.'/../fixtures/files/Metadata_test_file_w.jpg');
+    }
+
+    public static function tearDownAfterClass(): void
+    {
+        unlink(__DIR__.'/../fixtures/files/Metadata_test_file_w.jpg');
+    }
+
+    /**
+     * @covers MetadataManipulator::getReader
+     */
+    protected function setup(): void
+    {
+        try {
+            $this->service = static::getContainer()->get('metadata-manipulator');
+            $this->reader = $this->service->getReader();
+        }
+        catch(\Exception $e) {
+            // no-op
+        }
```

Comment:
This should be predictible. Or if it can fail, it should be in the test method itself